c/src/proactor/win_iocp.c

```diff
@@ -2145,8 +2132,8 @@ static void pconnection_tick(pconnection_t *pc) {
     if(!stop_timer(pc->context.proactor->timer_queue, &pc->tick_timer)) {
       // TODO: handle error
     }
-    uint64_t now = pn_i_now2();
-    uint64_t next = pn_transport_tick(t, now);
+    int64_t now = pn_proactor_now_64();
+    int64_t next = pn_transport_tick(t, now);
     if (next) {
       if (!start_timer(pc->context.proactor->timer_queue, &pc->tick_timer, tick_timer_cb, pc, next - now)) {
```

Comment:
Comment as above in the epoll case - but note that here there may actually be a bug as the subtraction is done using signed int64_t arithmetic which doesn't have defined wraparound semantics.